### PR TITLE
[#IOCIT-277] Added Ulid to definitions

### PR DESCRIPTION
**Pull request by michaeldisaro**
#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by AleDore — APPROVED** [review]

**Comment by michaeldisaro** [comment]
\azp run

File: .github/workflows/pr-title-linter-and-linker.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      - synchronize
+
+jobs:
+  lint:
+    name: Validate PR title And link Jira Issue
+    runs-on: ubuntu-22.04
+    env:
+      JIRA_COMMENT_REGEX: "^.*Jira.*"
+    steps:
+      - uses: Slashgear/action-check-pr-title@860e8dc639f8e60335a6f5e8936ba67ed2536890
```
Comment (BurnedMarshal) [inline .github/workflows/pr-title-linter-and-linker.yaml:18]: Why we are using the commit id instead the tag to refear the github action?

Comment (michaeldisaro) [reply .github/workflows/pr-title-linter-and-linker.yaml:18]: @AleDore said it was a security direction.

**Pull request merged**